rapidsai/values.yaml

```diff
@@ -6,15 +6,15 @@ dask:
     name: scheduler
     image:
       repository: "rapidsai/rapidsai"
-      tag: 0.15-cuda10.1-runtime-ubuntu18.04
+      tag: 0.17-cuda11.0-runtime-ubuntu18.04
       pullPolicy: IfNotPresent
       # See https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
       pullSecrets:
       #  - name: regcred
     replicas: 1
-    serviceType: "ClusterIP"
+    # serviceType: "ClusterIP"
     # serviceType: "NodePort"
-    # serviceType: "LoadBalancer"
+    serviceType: "LoadBalancer"
```

Comment:
We explicitly chose to make `ClusterIP` the default for security purposes. Having `LoadBalancer` as the default can result in the service being exposed to the internet. Therefore we should endeavour to make things secure by default.

Choosing `LoadBalancer` will likely be the most common use case, so perhaps we need to improve the documentation around this.

Reply:
updating the rapids.ai/cloud documentation to pass in ` --set dask.jupyter.serviceType="LoadBalancer"` while installing helm charts

Reply:
Perfect thanks!